jxlint/src/main/java/com/selesse/jxlint/model/ProgramOptions.java

```diff
@@ -69,6 +71,32 @@ public OutputType getOutputType() {
         }
     }
 
+    public String getSourcePathPrefix() {
+        String result;
+        if(options.containsKey(JxlintOption.SRC_PATH_PREFIX)) {
+            result = options.get(JxlintOption.SRC_PATH_PREFIX);
+        } else {
+            if(getSourceDirectory() == null) {
+                return "";
+            }
+            if (!options.containsKey(JxlintOption.WEB) && options.containsKey(JxlintOption.OUTPUT_TYPE_PATH)) {
```

Comment:
As far as I could test, `!options.containsKey(JxlintOption.WEB)` is wrong:

I wanted to express the condition "if not generated to be watched in the jetty web server", because in this case I want to have the full URI of the source directory.

This does not work, when I watch the report on the page `http://localhost:8380/report.jsp`:

* `options.containsKey(JxlintOption.WEB)` => false
* `options.get(JxlintOption.OUTPUT_TYPE_PATH)` => `C:\_____\AppData\Local\Temp\jxlint4995182164026642975.html`

So I get the relative path between this temporary file and the real file. My browser is opened on `http://localhost:8380/` page. Therefore the link does not work.

Even the link `file:/` might not be a good solution for direct live viewing, because I think that web browser do not accept to navigate to a `file:/` link when a page on `http://` page is opened.